docs/development/journalist_api.rst

```diff
@@ -103,21 +103,23 @@ The root endpoint describes the available resources:
 
 Response 200 (application/json):
 
-.. code:: sh
+.. code:: json
 
-  {
-    "current_user_url": "/api/v1/user",
-    "sources_url": "/api/v1/sources",
-    "submissions_url": "/api/v1/submissions",
-    "replies_url": "/api/v1/replies",
-    "token_url": "/api/v1/token"
-  }
+    {
+      "all_users_url": "/api/v1/users",
+      "auth_token_url": "/api/v1/token",
+      "current_user_url": "/api/v1/user",
+      "replies_url": "/api/v1/replies",
+      "seen_url": "/api/v1/seen",
+      "sources_url": "/api/v1/sources",
+      "submissions_url": "/api/v1/submissions"
+    }
 
-Sources ``[/sources]``
```

Comment:
I kind of liked having those endpoint paths in the headings, but we weren't consistent about it, so I think your cleanup is the right thing for now and we can potentially revisit later.